homeassistant/components/webostv/const.py

```diff
@@ -11,7 +11,7 @@
 PLATFORMS = [Platform.MEDIA_PLAYER]
 DATA_CONFIG_ENTRY = "config_entry"
 DATA_HASS_CONFIG = "hass_config"
-DEFAULT_NAME = "LG webOS Smart TV"
+DEFAULT_NAME = "LG webOS TV"
```

Comment:
This was changed since I noticed the device name in SSDP discovery does not include the word `Smart`, this is also what shown on the manufacturer app. I consider removing the term `smart` from the docs also